shared_model/backend/protobuf/transaction_responses/proto_unknown_tx_response.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_PROTO_UNKNOWN_TX_RESPONSE_HPP
+#define IROHA_PROTO_UNKNOWN_TX_RESPONSE_HPP
+
+#include "endpoint.pb.h"
+#include "interfaces/transaction_responses/unknown_tx_response.hpp"
+#include "utils/reference_holder.hpp"
+
+namespace shared_model {
+  namespace proto {
+    /**
+     * Transaction not found
+     */
+    class UnknownTxResponse final : public interface::UnknownTxResponse {
```

Comment:
I look on implementations of concrete tx responses, they very similar to each other. May we conduct template implementation of response classes?

Reply:
Also, I didn't find `toString` methods, may be better to add it?

Reply:
Template looks good but we can't define class with it :( or I don't really know how do you want to impl it

`toString` in `AbstractTxResponse`